synapse/handlers/room.py

```diff
@@ -776,15 +783,17 @@ async def create_room(
 
     async def _send_events_for_new_room(
         self,
-        creator,  # A Requester object.
-        room_id,
-        preset_config,
-        invite_list,
-        initial_state,
-        creation_content,
-        room_alias=None,
-        power_level_content_override=None,  # Doesn't apply when initial state has power level state event content
-        creator_join_profile=None,
+        creator: Requester,
+        room_id: str,
+        preset_config: str,
+        invite_list: List[str],
+        initial_state: StateMap,
+        creation_content: JsonDict,
+        room_alias: Optional[RoomAlias] = None,
+        power_level_content_override: Optional[
+            JsonDict
+        ] = None,  # Doesn't apply when initial state has power level state event content
```

Comment:
May be out of scope for this PR, but it'd be nicer to move this comment into the doc string. Then editors will display it.